### DISCUSSION TITLE
Status : Please report the error to the administrator, by sharing the file name.

### DISCUSSION THREAD

**Discussion by jfcouchot** [danymat/INSAnonym-utils#71] [Q&A, Bug report & Support]
Bonjour.
En local, toutes les métriques donnent un résultat.
Il nous est demandé de donner les noms des fichier 
femtorange1.zip et le nom du dataset est df_final.csv....
Merci pour votre aide.
JF

**Comment by danymat** [comment]
Bonjour, je ne comprends pas la question. Est-il possible d'expliciter un peu plus? Cordialement, Daniel

**Reply by beng-git** [reply]
Il y a une erreur sur la soumission #74

**Reply by jfcouchot** [reply]
Hello.
Sur le site de soumission :

#74 FemtOrange1 | -1 | -1 | Status : Please report the error to the administrator, by sharing the file name.
-- | -- | -- | --

Il nous est demandé de donner les noms des fichiers : 
femtorange1.zip et le nom du dataset est df_final.csv....
Merci pour votre aide.

**Reply by danymat** [reply]
Je viens de voir votre soumission, il faut passer del uniquement sur la colonne identifiants. Vous l'avez passée sur les colonnes longitude et latitude.

**Reply by nathanael-denis** [reply]
Bonjour, j'ai eu la même erreur lors de mes soumissions, mais probablement pas pour les mêmes raisons,
(avec deux messages auparavant erreur sur nom de fichier et un seul upload à la fois)

merci d'avance,

ND

**Reply by danymat** [reply]
> Bonjour, j'ai eu la même erreur lors de mes soumissions, mais probablement pas pour les mêmes raisons,
> (avec deux messages auparavant erreur sur nom de fichier et un seul upload à la fois)
> 
> merci d'avance,
> 
> ND

Est-il possible de me donner le numéro de la soumission?

**Reply by nathanael-denis** [reply]
Les soumissions #54 et #75 (ce sont les mêmes en pratiques, la seconde était pour réessayer)

**Reply by danymat** [reply]
Bonjour, n'hésitez pas a m'envoyer un mail pour que je puisse vous donner des informations confidentielles (daniel.mathiot@insa-cvl.fr)
